libs/sdk-common/src/buy/mod.rs

```diff
@@ -1,8 +1,10 @@
 use anyhow::Result;
+use async_trait::async_trait;
 
 pub mod moonpay;
 
-#[tonic::async_trait]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
```

Comment:
How does this work? Why does the future need `Send` in non-wasm, but not in wasm?

Reply:
Its a WASMism: https://github.com/rustwasm/wasm-bindgen/issues/2833